openpype/hosts/houdini/api/lib.py

```diff
@@ -835,6 +835,7 @@ def update_houdini_vars_context_dialog():
     update_vars = get_context_var_changes()
     if not update_vars:
         # Nothing to change
+        print("  - Nothing to change, Houdini Vars are up to date.")
```

Comment:
```suggestion
        log.debug("Nothing to change, Houdini vars are already up to date.")
```
_Assumes there's a `log` in this `lib.py` file already_

What's up with all the indentations and prefixed `  - ` in prints? :D Why is that needed? I see that a lot and never understood why.

Reply:
You're a mad man. Use `logging`.

Reply:
I can't see any printed messages whenever I use `log` in lib file :/ 
I can tell log is defined but it doesn't work

https://github.com/ynput/OpenPype/blob/e255c20c440211d3578fc7bcc7b350b6756dd859/openpype/hosts/houdini/api/lib.py#L21

Reply:
Let's stick to `print` (without these annoying indentations) till we know why logging not working in this scope.